azure-pipelines.yml

```diff
@@ -409,6 +409,23 @@ jobs:
         pythonVersion: '$(python.version)'
         profile: 'latest'
 
+- job: AutomationTest20200901
+  displayName: Automation Test (Profile 2020-09-01)
+  timeoutInMinutes: 120
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
+  steps:
+    - template: .azure-pipelines/templates/automation_test.yml
+      parameters:
+        pythonVersion: '$(python.version)'
+        profile: '2020-09-01-hybrid'
+
 - job: AutomationTest20190301
```

Comment:
Should we remove this?

Reply:
Th test folders will be added  @abbhar  can you pls conver this PR to draft?

Reply:
Should we remove the old profile AutomationTest20190301?

Reply:
I think we should not remove them because 2019-03-01-hybrid is still in use for CLI.

Reply:
Yes, we are fine with removing 2018 profile related tests, we should still have the 2019 profile tests

Reply:
This PR is already too many changes, i will create another PR for removing 2018 profile and corresponding tests.